content/en/dynamic_instrumentation/_index.md

```diff
@@ -114,27 +111,54 @@ For more information, read [How Dynamic Instrumentation Works][11].
 
 To create a metric probe:
 
-1. Complete the generic probe setup (choose service, environment, version, and probe location).
 1. Select **Metric** as the probe type.
+1. Complete the generic probe setup (choose service, environment, version, and probe location).
 1. Specify a name for the metric, which will be prefixed with `dynamic.instrumentation.metric.probe.`.
 1. Select a metric type (count, gauge, or histogram).
-1. Choose the value of the metric using the Debugger expression language. For count metrics this is optional, and if you omit it, every invocation increments the count by one.
+1. Choose the value of the metric using the [Dynamic Instrumenation expression language][15]. For count metrics this is optional, and if you omit it, every invocation increments the count by one.
 
 {{< img src="dynamic_instrumentation/metric_probe.png" alt="Creating a Dynamic Instrumentation metric probe" >}}
 
+### Creating a span probe
+
+A *span probe* emits a span when a method is executed. You can use a *span probe* as a more efficient alternative to [creating new spans with Custom Instrumentation]][13], as it does not require making code changes and redeploying your software. If the method throws an exception, the details of the exception will be associated with the newly created span's `error` tag.
```

Comment:
This doesn't render correctly: 
<img width="836" alt="Screenshot 2023-06-26 at 09 28 32" src="https://github.com/DataDog/documentation/assets/1101441/1de97c59-cace-4fdb-ae85-a2b5325ecf52">

Reply:
Fixed in f76d3d1908e29d59a9366e309684d8b810db3e6c